src/org/opendatakit/briefcase/model/BriefcasePreferences.java

```diff
@@ -181,4 +185,40 @@ public static HttpHost getBriefCaseProxyConnection() {
     }
     return null;
   }
+
+  /**
+   * Persist the user's decision to allow/disallow their behaviour being tracked.
+   * @param value (required) the boolean value representing the user's decision.
+   */
+  public static void setBriefcaseTrackingConsentProperty(Boolean value) {
+    if (value == null) {
+      Preference.APPLICATION_SCOPED.remove(BRIEFCASE_TRACKING_CONSENT_PROPERTY);
+    } else {
+      Preference.APPLICATION_SCOPED.put(BriefcasePreferences.BRIEFCASE_TRACKING_CONSENT_PROPERTY, value.toString());
+    }
+  }
+
+  /**
+   * Get the user's persisted decision to being tracked.
```

Comment:
Grammar: should be more a long the lines of "Get the user's persisted decision regarding their consent to being tracked".